README.md

```diff
@@ -675,6 +692,11 @@ public_key: 'PUBLIC KEY'
 private_key: 'PRIVATE KEY'
 ```
 
+## Caveats
+
+- Don't delete files until their `is_ready` flag is in `true` state (read:
```

Comment:
this seems to be REST API bug, we don't need this info here. It's better to fix the bug :)